bin/initPrism.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+PRISM_DIR="/usr/local/prism-src"
+PRISM_VERSION_TAG="v4.7"
+
+
+if [ -d "$PRISM_DIR" ]; then
+    cd "$PRISM_DIR"; git pull;
+else
+    echo "Using sudo to install prism to $PRISM_DIR"
+    sudo bash -c "mkdir '$PRISM_DIR' && chmod 777 '$PRISM_DIR'"
```

Comment:
Additionally, if we're explicitly invoking a shell from a script, vanilla `sh` is generally the safest way to go. `bash` is less portable than you may think it is and even though `mkdir` and `chmod` will work in it, perhaps the entire script should be run in `sh`...you know, since some systems may not even have `bash` at all. `sh`, like `nano` is the lowest common denominator that all systems have